dropwizard-jetty/src/main/java/io/dropwizard/jetty/NetUtil.java

```diff
@@ -0,0 +1,99 @@
+ * It has been modified, to remove dependencies on other classes, and to convert to methods, rather than a
+ * static value.
+ */
+public class NetUtil {
+    public static final int DEFAULT_TCP_BACKLOG_WINDOWS = 200;
+    public static final int DEFAULT_TCP_BACKLOG_LINUX = 128;
+    public static final String TCP_BACKLOG_SETTING_LOCATION = "/proc/sys/net/core/somaxconn";
+
+
+
+    /**
+     * The SOMAXCONN value of the current machine.  If failed to get the value,  {@code 200}  is used as a
+     * default value for Windows or {@code 128} for others.
+     */
+    public static int getTcpBacklog() {
+        return getTcpBacklog(getDefaultTcpBacklog());
+    }
+
+    /**
+     * The SOMAXCONN value of the current machine.  If failed to get the value, <code>defaultBacklog</code> argument is
+     * used
+     */
+    public static int getTcpBacklog(int tcpBacklog) {
+        // Taken from netty.
+
+        // As a SecurityManager may prevent reading the somaxconn file we wrap this in a privileged block.
+        //
+        // See https://github.com/netty/netty/issues/3680
+        return AccessController.doPrivileged(new PrivilegedAction<Integer>() {
+            @Override
+            public Integer run() {
+                // Determine the default somaxconn (server socket backlog) value of the platform.
+                // The known defaults:
+                // - Windows NT Server 4.0+: 200
+                // - Linux and Mac OS X: 128
+                int somaxconn = tcpBacklog;
+                File file = new File(TCP_BACKLOG_SETTING_LOCATION);
+                BufferedReader in = null;
+                try {
+                    // file.exists() may throw a SecurityException if a SecurityManager is used, so execute it in the
+                    // try / catch block.
+                    // See https://github.com/netty/netty/issues/4936
+                    if (file.exists()) {
+                        in = new BufferedReader(new InputStreamReader(new FileInputStream(file), Charset.forName("UTF-8")));
+                        somaxconn = Integer.parseInt(in.readLine());
+                    }
+                } catch (SecurityException | IOException | NumberFormatException e) {
+                    // file.exists() may throw a SecurityException, in this
+                    // case we are just returning the default somaxconn that was passed in.
+                } finally {
+                    if (in != null) {
+                        try {
+                            in.close();
+                        } catch (Exception e) {
+                            // Ignored.
+                        }
+                    }
+                }
+                return somaxconn;
+            }
+        });
+
+    }
+
+    public static boolean isWindows() {
+        boolean windows = System.getProperty("os.name", "").toLowerCase(Locale.US).contains("win");
+        return windows;
+    }
+
+    public static int getDefaultTcpBacklog() {
```

Comment:
I'll let @joschi make the call. It's the same implementation as [netty's](https://github.com/netty/netty/blob/56a2f64665494ee0d52fba7f391de0bea6144e29/common/src/main/java/io/netty/util/internal/PlatformDependent.java#L798-L804), so I'm ok with either.